pinot-core/src/main/java/com/linkedin/pinot/core/query/selection/SelectionOperatorUtils.java

```diff
@@ -420,9 +421,25 @@ public static DataTable getDataTableFromRows(@Nonnull Collection<Serializable[]>
   @Nonnull
   public static SelectionResults renderSelectionResultsWithoutOrdering(@Nonnull List<Serializable[]> rows,
       @Nonnull DataSchema dataSchema, @Nonnull List<String> selectionColumns) {
+    // TODO: remove the code for backward compatible after server updated to the latest code.
```

Comment:
All these schema compatible merge code has not been released. When that is released, I can make a ticket with the version number.